modeling-cmds/src/def_enum.rs

```diff
@@ -40,6 +40,18 @@ define_modeling_cmd_enum! {
             Vec::new()
         }
 
+        /// Evaluates the position of a path in one shot (engine utility for kcl executor)
+        #[derive(
+            Debug, Clone, Serialize, Deserialize, JsonSchema, ModelingCmdVariant,
+        )]
+        pub struct EngineUtilEvaluatePath {
+            /// The path in json form (the serialized result of the kcl Sketch/Path object
+            pub path_json: String,
+
+            /// The evaluation parameter
```

Comment:
Could you give a bit more detail please? These docs explain how to use the endpoint, and right now I don't know how to use the endpoint.

Reply:
this is the exact "parameter" that i was talking about during my curves/nurbs talk at kitty camp.  its not a programming parameter, its a curve parameter.   I suppose we could say "path curve parameter?"